tests/conftest.py

```diff
@@ -139,6 +154,7 @@ def user_password() -> str:
 @pytest.fixture()
 def user(app: Flask, user_password: str, database: str) -> User:
     user = User(password=user_password)
+    user.tier_id = 1
```

Comment:
Set this via the constructor?

Reply:
We actually want tier_id to start out as None by default, because on first login, if it's None, it displays the tier selection page. But in the tests, I wanted them all to start out on the free tier, to skip the selection page.

Reply:
Oh sorry I meant just `User(password=user_password, tier_id=1)`